From bbde75d71b51b6b60d2a672b94a67b856df93d25 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Tue, 16 Oct 2018 13:57:38 +0300
Subject: [PATCH 328/706] xgmac_mdio: Add license and copyright

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit ecf34972ed7220b27bc40e3c5249195741f4b87e)
(cherry picked from commit 2cd4d5e59c87c69f5c26c82903ef8e09a47ef743)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/xgmac_mdio.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/net/ethernet/freescale/xgmac_mdio.h b/drivers/net/ethernet/freescale/xgmac_mdio.h
index eb4ce4e23a3d..9a9bb7932716 100644
--- a/drivers/net/ethernet/freescale/xgmac_mdio.h
+++ b/drivers/net/ethernet/freescale/xgmac_mdio.h
@@ -1,3 +1,6 @@
+/* SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause) */
+/* Copyright 2018 NXP */
+
 #include <linux/kernel.h>
 #include <linux/mdio.h>
 
-- 
2.17.1

